@navikt/core/react/src/date/datepicker/DatePicker.tsx

```diff
@@ -88,8 +88,8 @@ export const DatePicker = forwardRef<HTMLDivElement, DatePickerProps>(
     const ariaId = useId(id);
     const [open, setOpen] = useState(_open ?? false);
 
-    const wrapperRef = useRef<HTMLDivElement | null>(null);
-    const mergedRef = useMergeRefs(wrapperRef, ref);
+    const [wrapperRef, setWrapperRef] = useState<HTMLDivElement | null>(null);
```

Comment:
Consider adding a comment explaining why useState is used here instead of useRef,
and/or creating a story where open is initially true.